src/common/dom/dynamic-content-directive.ts

```diff
@@ -1,7 +1,11 @@
 import { directive, Part, NodePart } from "lit-html";
 
 export const dynamicContentDirective = directive(
-  (tag: string, properties: { [key: string]: any }) => (part: Part): void => {
+  (
+    tag: string,
+    properties?: { [key: string]: any },
+    attributes?: { [key: string]: any }
```

Comment:
Let's remove it for now, you can set ID as a prop. If we do end up adding it, you should name it `initAttributes`. As right now, they are only set on initialization.